src/config/user.cr

```diff
@@ -48,7 +54,7 @@ struct DppmRestApi::Config::User
   end
 
   def self.deserialize(groups : String)
-    groups.split(',').map { |id| id.to_i base: 16 }
+    groups.split(',').map { |id| id.to_i base: 16 }.to_set
```

Comment:
An intermediary `Array` can be avoider

Reply:
... but apparently shouldn't be. :thinking: 

https://github.com/DFabric/dppm-rest-api/commit/d1e005629bd7bacaacbeaa55043987aabf7f3415?diff=unified

Reply:
No, declare a `Set` then add each `id.to_i` to it.

Reply:
`groups.split(',') { ... }`

Reply:
Oh, I see, I forgot that `String#split` had a block variant.

```
                         iterator  43.56k ( 22.96µs) (± 4.78%)  10.7kB/op   1.23× slower
               intermediate array  45.50k ( 21.98µs) (± 3.46%)  11.5kB/op   1.18× slower
with String#split's block variant  53.78k ( 18.60µs) (± 4.60%)  7.06kB/op        fastest
```